metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
I would be with @jrleeman , SB, ML, MU are common field names in web-based visualization, see convective products [here:](http://weather.cod.edu/forecast/) and severe tab [here,](https://ensemble.ucar.edu/images.php?d=2017072500&f=ref1km_stamp&r=NW&i=1) but I have seen it differently elsewhere (see severe weather tab [here.](http://www.pivotalweather.com/model.php)) I think users should be able to figure it out either way since those are both fairly conventional naming methods.